Converted ZOHOCRMSDK-2.1.csproj to netstandard2.0 (from NetFramework4.6.1)

petrechitashvili opened the pull request:
Converted ZOHOCRMSDK-2.1.csproj to the SDK style & netstandard2.0 (from NetFramework4.6.1) _
Removed packages.config, MySql.Data & DBStore.cs files to make the project DB-agnostic _
Removed 'private' access modifier (default = private) _
Removed 'this' keyword wherever appropriate _
Introduced Initializers { item1, item2 } instead of Adding individual items (to Lists).

petrechitashvili commented:
@raja-7453 @Aswin97 @pravesh-a @Rishabh-Raghunath Greetings 👋🏼

I hope you don't mind me creating a Pull Request.

Have a good day!
- Petre

raja-7453 commented:
Thanks, @petrechitashvili  for your contribution. 
- We do not provide netstandard2.0 as it is not supported in older versions of Visual Studio (before 2017).
- There is an option for custom persistence provided in our SDKs, where implementation can be done using the TokenStore interface which will be suitable for any database.
- Kindly let us know the reason why **private** modifiers and **this** keyword were removed.
- Regarding the Initializers, we do have plans to provide it as a list in future versions

petrechitashvili commented:
> Thanks, @petrechitashvili for your contribution.
> 
> * We do not provide netstandard2.0 as it is not supported in older versions of Visual Studio (before 2017).
> * There is an option for custom persistence provided in our SDKs, where implementation can be done using the TokenStore interface which will be suitable for any database.
> * Kindly let us know the reason why **private** modifiers and **this** keyword were removed.
> * Regarding the Initializers, we do have plans to provide it as a list in future versions

Hey Raja,

Sure thing.

For those using Visual Studio *after* 2019 - Here's a Fork from the main repo, converted to netstandard2.0, cleaned not to contain irrelevant packages - https://github.com/AppifySheets/zohocrm-csharp-sdk-2.1-netstandard2.0

Have a good day!
- Petre



I leave the answers to your questions will be answered by a higher entity 😎
